src/library_webgl.js

```diff
@@ -1307,6 +1307,17 @@ var LibraryGL = {
 #if MAX_WEBGL_VERSION >= 2
               case 0x8919: // GL_SAMPLER_BINDING
               case 0x8E25: // GL_TRANSFORM_FEEDBACK_BINDING
+              //buffer object binding
+              case 0x88ED: // GL_PIXEL_PACK_BUFFER_BINDING
+              case 0x88EF: // GL_PIXEL_UNPACK_BUFFER_BINDING
+              case 0x8C8F: // GL_TRANSFORM_FEEDBACK_BUFFER_BINDING
+              case 0x8A28: // GL_UNIFORM_BUFFER_BINDING
+              case 0x8F36: // COPY_READ_BUFFER_BINDING, same with COPY_READ_BUFFER
+              case 0x8F37: // COPY_WRITE_BUFFER_BINDING, same with COPY_WRITE_BUFFER
```

Comment:
nit: it's not relevant that these have the same values as other enums; glGet by spec takes particular enums.
Also nit: missing `GL_`

```suggestion
              case 0x8F36: // GL_COPY_READ_BUFFER_BINDING
              case 0x8F37: // GL_COPY_WRITE_BUFFER_BINDING
```

Reply:
Got it. I'll remove these two comments